stdlib/out_channel.mli

```diff
@@ -16,6 +16,8 @@
 (** Output channels.
 
     This module provides functions for working with output channels.
+    For example, you can use this module to print to the terminal or
+    write to a file.
```

Comment:
You should keep the same order between `In_channel` and `Out_channel` , and I feel that the order is  "do I/O on a file" then "do I/O on stdin/stdout" is better.